### DISCUSSION TITLE
Update Image.

### DISCUSSION THREAD

**Discussion by eelgaev** [emilk/egui#3745] [Q&A]
Hey guys, I am enjoying egui on multiple projects so far, I was able to do many things on such low memory footprint and impressive performance. In my latest project I came in a bit of an issue with `egui::Image`, for some reason, it is impossible to update an existing Image.

```rust
            ui.add(egui::Image::from_bytes(
                "bytes://controller.svg".to_owned(),
                self.controller_ui.get_as_bytes(),
            ));
```

In this code I am trying to display an svg drawing that is sometimes updated, but for some reason, once it's called, it cannot update. I guess that it caches the last image under that name and won't write again. I am worried to create new instances by using new name, what should I do? is it possible to clear the cache when I am sure I have an update?

**Comment by rustbasic** [comment]
use ctx.forget_image()

example)
```
    let uri = image.source().uri().unwrap_or_default().to_string();
    ui.ctx().forget_image(&uri);
```
